.gitignore

```diff
@@ -10,4 +10,5 @@ caches/*
 akara.ini
 akara.conf
 akara.confc
-ENV/*
\ No newline at end of file
+ENV/*
+couchdb.*
```

Comment:
IIRC CouchDB logs are generated in the current working directory when you start it as a background process with `couchdb -b`.